### PR TITLE
SQL URL query parameters

### PR DESCRIPTION
**Pull request by ricwo**
**Proposed changes**:
- add query parameters to SQL connection URL

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: docs/api/tracker-stores.rst [hunk 1]
```diff
@@ -57,13 +59,16 @@ SQLTrackerStore
 
             rasa run -m models --endpoints endpoints.yml
 :Parameters:
+    - ``domain`` (default: ``None``): Domain object associated with this tracker store
```
[the remark this answers is not in the document]
Comment (ricwo) [reply docs/api/tracker-stores.rst]: just whatever domain is currently active in `server` or `run`. We initialise with `None` but update it once a model is loaded:
https://github.com/RasaHQ/rasa/blob/master/rasa/core/agent.py#L322

File: docs/api/tracker-stores.rst [hunk 2]
```diff
@@ -57,13 +59,16 @@ SQLTrackerStore
...
     - ``dialect`` (default: ``sqlite``): The dialect used to communicate with your SQL backend.  Consult the `SQLAlchemy docs <https://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls>`_ for available dialects. 
-    - ``url`` (default: ``None``): URL of your SQL database
+    - ``host`` (default: ``None``): URL of your SQL server
+    - ``port`` (default: ``None``): Port of your SQL server
     - ``db`` (default: ``rasa.db``): The path to the database to be used
     - ``username`` (default: ``None``): The username which is used for authentication
     - ``password`` (default: ``None``): The password which is used for authentication
-    - ``collection`` (default: ``conversations``): The collection name which is
-      used to store the conversations
+    - ``event_broker`` (default: ``None``): Event broker to publish events to
```
[the remark this answers is not in the document]
Comment (ricwo) [reply docs/api/tracker-stores.rst]: yes events are published out of the tracker store's `save()` method

File: docs/api/tracker-stores.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (akelad) [reply docs/api/tracker-stores.rst]: ah ofc

File: docs/api/tracker-stores.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (akelad) [reply docs/api/tracker-stores.rst]: oh ok - what's the difference between passing that in the `endpoints.yml` and putting it in there btw?

**Pull request merged**